christfellowshipchurch/apollos-app
# Add chat toggle in landscape for live streams

This PR adds the ability to toggle chat in landscape mode. Turn your simulator landscape on a live stream video. Click the chat icon in the top right. 

TO TEST:

Toggle chat, switch between landscape and portrait, minify video component and make sure the video player mini-controls look correct in all modes. All feedback is welcome including design suggestions.

![image](https://user-images.githubusercontent.com/2528817/96489958-c5f16480-1205-11eb-8293-5364d58400a8.png)
![image](https://user-images.githubusercontent.com/2528817/96486255-66458a00-1202-11eb-823a-f1c91374a87a.png)
